src/modes/base.mode.ts

```diff
@@ -196,4 +196,13 @@ export abstract class TerraDrawBaseDrawMode<T extends CustomStyling> {
 			return value;
 		}
 	}
+
+	/**
+	 * Sets the coordinate precision for the mode.
+	 *
+	 * @param {number} coordinatePrecision The precision value for the coordinates.
+	 */
+	public setCoordinatePrecision(coordinatePrecision: number) {
+		this.coordinatePrecision = coordinatePrecision;
+	}
```

Comment:
I think it may be best to remove this function and just set this via the `register` function on line 114

Reply:
Ah sorry i forgot to remove this function.
I will update the pr and test it tomorrow. 👍🏻